HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -482,6 +482,13 @@ public IActionResult HouseholdIncome_Post(HouseholdIncomeViewModel viewModel)
     public IActionResult CheckAnswers_Get()
     {
         var questionnaire = questionnaireService.GetQuestionnaire();
+
+        if (questionnaire.UneditedData is not null)

```

Comment:
Please add a comment here explaining what this is doing - I know it's been copied from SEA, but this logic doesn't really belong in the controller.

Reply:
Sure, I can also move it into a method in `QuestionnaireService` if you think that's better?

Reply:
Ideally I think it would live there, but the comment is sufficient